.pre-commit-config.yaml

```diff
@@ -53,7 +53,6 @@ repos:
   - id: check-json
   - id: check-merge-conflict
   - id: check-xml
-  - id: check-yaml
```

Comment:
What is this and what does it have to do with a `shell.nix` PR?

Reply:
this became wip and i will squish cleanup after i got everything running. this on slapped in because of pre-commit hook was borked for reasons